src/move/econia/sources/market.move

```diff
@@ -2766,13 +2903,16 @@ module econia::market {
                         fill_size, complete_fill, optional_base_coins,
                         quote_coins, fill_size * lot_size,
                         ticks_filled * tick_size);
-                // Get taker events handle.
-                let taker_handle = &mut order_book_ref_mut.taker_events;
-                // Emit corresponding taker event.
-                event::emit_event(taker_handle, TakerEvent{
-                    market_id, side, market_order_id, maker, custodian_id:
-                    maker_custodian_id, size: fill_size, price});
```

Comment:
Backwards compatibility means that anyone who had built anything on top of a previous version of this code would not have to make any changes for their code to keep on running as expected.

As we are no longer emitting taker events, and any system that relied on taker events would have to make the switch over to fills events, this change is not backwards compatible.

Reply:
@bogberry Good point, backwards-compatibility is only in the sense that the contract itself will pass a bytecode verifier upgrade check, required to deploy to the same address on mainnet

Tagging #321 to incorporate refactors on that iteration

Reply:
Yes, so it's not really backward-compatible. Aptos simply checks the struct declarations and function signatures

Reply:
Yes, so it's not really backward-compatible. Aptos simply performs a minimal check where the struct declarations and function signatures are compared to catch any changes that are obviously not backwards compatible, as far as I'm aware. This is in no way a guarantee of backwards compatibility, and since we know that this in fact isn't a backwards compatible change, we should stop describing it as such.